mds/apis/agency_api/v0_x/vehicles.py

```diff
@@ -110,12 +125,23 @@ class DeviceTelemetrySerializer(serializers.Serializer):
     timestamp = utils.UnixTimestampMilliseconds(
         help_text="Unix timestamp in milliseconds"
     )
+    charge = serializers.FloatField(
+        required=False,
+        source="battery_pct",
+        min_value=0,
+        max_value=1,
+        help_text="Percent battery charge of vehicle, expressed between 0 and 1",
+    )
 
 
 class DeviceEventSerializer(serializers.Serializer):
     event_type = serializers.ChoiceField(
         choices=enums.choices(enums.EVENT_TYPE), help_text="Vehicle event."
     )
+    # Defined in MDS spec, but useless for us since we store the latest event
```

Comment:
I don't understand the comment : this should feed the model `timestamp` field instead of `telemetry.timestamp`. in the `create` method L.159.